transport/tchannel/channel_outbound.go

```diff
@@ -156,10 +156,13 @@ func (o *ChannelOutbound) Call(ctx context.Context, req *transport.Request) (*tr
 		return nil, toYARPCError(req, err)
 	}
 
-	// Inject tracing system baggage
-	reqHeaders := tchannel.InjectOutboundSpan(call.Response(), req.Headers.Items())
-
-	if err := writeRequestHeaders(ctx, format, reqHeaders, call.Arg2Writer); err != nil {
+	reqHeaders := req.Headers.Items()
+	if o.transport.originalHeader {
+		reqHeaders = req.Headers.OriginalItems()
+	}
+	// We do not want to canonicalize these headers as channel_outbound is deprecated
```

Comment:
the reason is because the baggage headers are transport implementation details that are stripped out (and stored in the context), so users don't interact with it, so shouldn't need to canonicalize it.